src/main/java/com/google/netpcapanalysis/dao/BucketDaoImpl.java

```diff
@@ -0,0 +1,228 @@
+package com.google.netpcapanalysis.dao;
+
+import com.google.netpcapanalysis.models.Flagged;
+import com.google.netpcapanalysis.models.PCAPdata;
+import java.util.*; 
+import java.io.*;
+import java.lang.*;
+import java.util.Map.Entry;
+
+import io.pkts.PacketHandler;
+import io.pkts.Pcap;
+import io.pkts.buffer.Buffer;
+import io.pkts.packet.Packet;
+import io.pkts.packet.TCPPacket;
+import io.pkts.packet.UDPPacket;
+import io.pkts.packet.IPPacket;
+import io.pkts.protocol.Protocol;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
```

Comment:
removed imports that aren't needed.

Reply:
Eclipse has an easier way to clean up unused imports but you can also use a script like this if you dont use an IDE: https://gist.github.com/rodrigosetti/4734557